system-int-test/src/cmr/system_int_test/utils/ingest_util.clj

```diff
@@ -585,22 +588,23 @@
                            :short-name short-name
                            :cmr-only cmr-only
                            :small small})
-     (echo-util/create-providers (s/context) {provider-guid provider-id})
+     ;; Create provider in mock echo with the guid set to the ID to make things easier to sync up
+     (echo-util/create-providers (s/context) {provider-id provider-id})
 
      (when grant-all-search?
        (echo-util/grant (s/context)
                         [echo-util/guest-read-ace
                          echo-util/registered-user-read-ace]
-                        :catalog-item-identity
-                        (assoc (echo-util/catalog-item-id provider-guid)
-                               :collection-applicable true
-                               :granule-applicable true)))
+                        :catalog_item_identity
+                        (assoc (echo-util/catalog-item-id provider-id)
+                               :collection_applicable true
+                               :granule_applicable true)))
      (when grant-all-ingest?
-       (echo-util/grant-all-ingest (s/context) provider-guid))
+       (echo-util/grant-all-ingest (s/context) provider-id))
```

Comment:
Its already changed so I don't think its all that relevant anymore but all these functions should have stayed the same in regards to provider-guid, instead looking up the provider id if possible.  It would have prevented a lot of these changes.